dev-util/shellcheck/shellcheck-0.8.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.7
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+MY_PN="ShellCheck"
+MY_P="${MY_PN}-${PV}"
+S="${WORKDIR}/${MY_P}"
+
+DESCRIPTION="Shell script analysis tool"
+HOMEPAGE="https://www.shellcheck.net/"
+SRC_URI="https://hackage.haskell.org/package/${MY_P}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-haskell/aeson:=[profile?]
+	>=dev-haskell/diff-0.2.0:=[profile?]
+	>=dev-haskell/mtl-2.2.1:=[profile?]
+	>=dev-haskell/parsec-3.0:=[profile?]
+	>=dev-haskell/quickcheck-2.7.4:2=[profile?]
```

Comment:
Without `dev-haskell/quickcheck[template-haskell]` there is an error during build:

```
[ 5 of 23] Compiling ShellCheck.Fixer ( src/ShellCheck/Fixer.hs, dist/build/ShellCheck/Fixer.o, dist/build/ShellCheck/Fixer.dyn_o )

src/ShellCheck/Fixer.hs:409:12: error:
    • Variable not in scope:
        quickCheckAll
          :: template-haskell-2.16.0.0:Language.Haskell.TH.Lib.Internal.ExpQ
    • Perhaps you meant ‘quickCheck’ (imported from Test.QuickCheck)
    |
409 | runTests = $quickCheckAll
    |            ^^^^^^^^^^^^^^
[ 6 of 23] Compiling ShellCheck.Data  ( src/ShellCheck/Data.hs, dist/build/ShellCheck/Data.o, dist/build/ShellCheck/Data.dyn_o )
[15 of 23] Compiling ShellCheck.ASTLib ( src/ShellCheck/ASTLib.hs, dist/build/ShellCheck/ASTLib.o, dist/build/ShellCheck/ASTLib.dyn_o )

src/ShellCheck/ASTLib.hs:768:12: error:
    • Variable not in scope:
        quickCheckAll
          :: template-haskell-2.16.0.0:Language.Haskell.TH.Lib.Internal.ExpQ
    • Perhaps you meant ‘quickCheck’ (imported from Test.QuickCheck)
    |
768 | runTests = $quickCheckAll
    |            ^^^^^^^^^^^^^^
```